src/pykeen/trackers/tensorboard.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+"""An adapter for TensorBoard."""
+
+import pathlib
+import time
+from typing import Any, Dict, Mapping, Optional, TYPE_CHECKING, Union
+
+from .base import ResultTracker
+from ..constants import PYKEEN_LOGS
+from ..utils import flatten_dictionary
+
+if TYPE_CHECKING:
+    import torch.utils.tensorboard
+
+__all__ = [
+    'TensorBoardResultTracker',
+]
+
+
+class TensorBoardResultTracker(ResultTracker):
+    """A tracker for TensorBoard."""
+
+    summary_writer: 'torch.utils.tensorboard.SummaryWriter'
+    path: pathlib.Path
+
+    def __init__(
+        self,
+        experiment_path: Union[None, str, pathlib.Path] = None,
+        experiment_name: Optional[str] = None,
+        tags: Optional[Dict[str, Any]] = None,
+    ):
+        """
+        Initialize result tracking via Tensorboard.
+
+        :param experiment_path:
+            The experiment path. A custom path at which the tensorboard logs will be saved.
+        :param experiment_name:
+            The name of the experiment, will be used as a sub directory name for the logging. If no default is given,
+            the current time is used. If set, experiment_path is set, this argument has no effect.
+        :param tags:
+            The additional run details which are presented as tags to be logged
+        """
+        import torch.utils.tensorboard
+        self.tags = tags
```

Comment:
Everything looks good to go on this PR except the tags. they appear to be ignored. Do you know where these should be used @sbonner0?

Reply:
Hi @cthoyt, I think these tags are just a hangover from when I used the mlflow file as a template. I'll remove them.